src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/removepolicy/RemovePolicyResponse.kt

```diff
@@ -0,0 +1,44 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.indexstatemanagement.transport.action.removepolicy
+
+import com.amazon.opendistroforelasticsearch.indexmanagement.indexstatemanagement.util.FailedIndex
+import com.amazon.opendistroforelasticsearch.indexmanagement.indexstatemanagement.util.UPDATED_INDICES
+import com.amazon.opendistroforelasticsearch.indexmanagement.indexstatemanagement.util.buildInvalidIndexResponse
+import org.elasticsearch.action.ActionResponse
+import org.elasticsearch.common.io.stream.StreamInput
+import org.elasticsearch.common.io.stream.StreamOutput
+import org.elasticsearch.common.xcontent.ToXContent
+import org.elasticsearch.common.xcontent.ToXContentObject
+import org.elasticsearch.common.xcontent.XContentBuilder
+import java.io.IOException
+
+class RemovePolicyResponse : ActionResponse, ToXContentObject {
```

Comment:
I believe we have a few APIs that return this updated, failed, failures response format. Should we have a generic response that can be reused for them instead? Otherwise seems like you'll end up creating this exact response for add, remove, change, retry

Reply:
will refactor